src/decoder/image.rs

```diff
@@ -562,18 +567,25 @@ impl Image {
                 Predictor::Horizontal => {
                     return Err(TiffError::UnsupportedError(
                         TiffUnsupportedError::HorizontalPredictor(color_type),
-                    ))
+                    ));
                 }
                 Predictor::FloatingPoint => {
                     return Err(TiffError::UnsupportedError(
                         TiffUnsupportedError::FloatingPointPredictor(color_type),
                     ));
                 }
             },
+            (
+                ColorType::Multiband {
+                    bit_depth: _n,
+                    num_samples: _samples,
+                },
+                _,
+            ) => {}
```

Comment:
A little unsure if this should be a separate branch, or go in one of the branches above. Not familiar with what the Horizontal and FloatingPoint predictors are.

Reply:
The predictors are filters that are applied to the pixel data before compression, to help improve the compression ratio. Multiband should be included in both the match existing match arms where Gray is.

Reply:
Ok, done at e0b9ae040fe08842edc8f58182f89a31aa0c0338, but best if you can double check it. I don't have an multi-band TIFF file with a Horizontal/FloatingPoint predictor to test this out on.